README.md

```diff
@@ -145,7 +146,7 @@ Follow these steps to setup documenso on you local machine:
   git clone https://github.com/documenso/documenso
   ```
 - Run `npm i` in root directory
-- Rename `.env.example` to `.env`
+- Now run `cp .env.example .env` to create a copy of .env file.
```

Comment:
It's a good idea to keep the original `.env.example` for backup purposes. Incase something goes wrong, the original `.env.example` will be helpful.